Source/CarthageKit/Project.swift

```diff
@@ -635,7 +635,8 @@ public final class Project { // swiftlint:disable:this type_body_length
 		zipFile: URL,
 		projectName: String,
 		pinnedVersion: PinnedVersion,
-		toolchain: String?
+		toolchain: String?,
+        platforms: [Platform]
```

Comment:
Carthage uses tabs instead of spaces for indentation. Could you please use tabs for your changes?

Reply:
🤦‍♂️
Fix this commit 
be3ecaf